.github/workflows/windows-qt6.yml

```diff
@@ -50,6 +50,10 @@ jobs:
         uses: hendrikmuhs/ccache-action@v1.2
         with:
           max-size: 1G
+          key: build-ccache-win64-qt6-${{ github.event.pull_request.base.ref || github.ref_name }}
```

Comment:
```suggestion
          key: build-ccache-win64-qt6-${{ github.event.pull_request.base.ref || github.ref_name }}-${{ github.run_id }}
```

Reply:
By default hendrikmuh appends a timestamp, I guess that's sufficient, but the additional run_id won't hurt

Reply:
oh so then you don't need to specify the restore key I guess (if you don't put the run.id).